### PR TITLE
Feature/splash

### PR DESCRIPTION
**Pull request by Manuel-Mantilla**
-A new image loaded inside launch screen for a splash animation (animation in progress)
-A new logo for launching the app with a different look

https://github.com/SSPF20/RickAndMortyAPI/assets/92768503/62cea01b-9511-45a2-9c42-edfdc0e9602d

### CODE REVIEW

**Comment by hgp691** [comment]
What do you mean with "Animation in progress"?

File: RickAndMorty/AppDelegate.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        Thread.sleep(forTimeInterval: 1)
```
Comment (hgp691) [inline RickAndMorty/AppDelegate.swift]: Why we need this?

**Comment by hgp691** [comment]
@Manuel-Mantilla  Any updates on this PR ?

File: RickAndMorty.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -14,8 +14,9 @@
 		357802C42A5CE2E9009FE1D0 /* RMCharacterCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 357802C32A5CE2E9009FE1D0 /* RMCharacterCellViewModel.swift */; };
 		3578B1722A3BC689008AA38A /* RMCharacterStatusView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3578B1712A3BC689008AA38A /* RMCharacterStatusView.swift */; };
 		35B5CCB52A85C5F900B11614 /* SDWebImageSwiftUI in Frameworks */ = {isa = PBXBuildFile; productRef = 35B5CCB42A85C5F900B11614 /* SDWebImageSwiftUI */; };
-		35F34D272A3D6C3300F4FAAE /* RMCharacterImageSpecieView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 35F34D262A3D6C3300F4FAAE /* RMCharacterImageSpecieView.swift */; };
+		35F34D272A3D6C3300F4FAAE /* RMCharacterSpecieView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 35F34D262A3D6C3300F4FAAE /* RMCharacterSpecieView.swift */; };
```
Comment (hgp691) [inline RickAndMorty.xcodeproj/project.pbxproj:17]: Why are changes in this part?

**Comment by SSPF20** [comment]
There are conflicting files to be solved in the branch

**Pull request closed without merging**